treeherder/config/settings.py

```diff
@@ -505,53 +505,42 @@ def server_supports_tls(url):
 PULSE_DATA_INGESTION_SOURCES = env.json(
     "PULSE_DATA_INGESTION_SOURCES",
     default=[
-        # {
-        #     "name": "exchange/taskcluster-treeherder/v1/jobs",
-        #     "projects": [
-        #         'mozilla-central',
-        #         'mozilla-inbound'
-        #         # other repos TC can submit to
-        #     ],
-        #     "destinations": [
-        #         'production'
-        #         'staging'
-        #     ]
-        # },
-        # {
-        #     "name": "exchange/treeherder-test/jobs",
-        #     "projects": [
-        #         'mozilla-inbound'
-        #     ],
-        #     "destinations": [
-        #         'production'
-        #         'staging'
-        #     ]
-        #
-        # }
+        {
+            "exchange": "exchange/taskcluster-treeherder/v1/jobs",
+            "projects": [
+                'mozilla-central.#',
```

Comment:
Actually, ``projects`` should be:

    "projects": [
        '#',
    ],

So that we get all the projects by default.  Destination should probably also just be ``'#'``.

Reply:
Cool, I will push another commit for this tweak before squashing.

Reply:
Oh, I remembered had rebased to get my local npm/yarn happy after other changes on master. I'm going to force push all 3 commits here, but they are new SHA's.